maths/cube_root.py

```diff
@@ -0,0 +1,10 @@
+
+def cube_root(a: float) ->float:
+    """
+    it just give cube root
+    """
+    return a ** ( 1 / 3 )
+
+if __name__ == "__main__":
+    """just for testing"""
+    print(cube_root(27))
```

Comment:
```suggestion
    print(cube_root(27))
    
```
PEP 8 fix (file must end with a blank line)